CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+function(testFileExists fileName)
+
+   if (NOT EXISTS ${fileName})
+      message(FATAL_ERROR "Could not find file " ${fileName})
```

Comment:
Perhaps not at the global level, but certainly at the point where this is used, we should print a more detailed message like "Did you use 'git clone --recursive' to checkout all sub-repositories".